testing/test_dataset.py

```diff
@@ -1520,22 +1520,22 @@ def testArrayMinMeanMax(self):
     @pytest.mark.parametrize('xform',
                              [False, True],
                              ids=['Original Values', 'XFormed Values'])
-    def testArrayMMMShape(self, S5E25D40Shapes, S5E25D40MMMs, XformS5E25D40MMMs, xform):
+    def testArrayMMMShape(self, S5E25D40Shapes, S5E25D40MMMs, XformS5E25D40MMMs, xform: bool):
         """ Test that the Xformed or Original dict of arrayMinMeanMax returns has the right shape """
 
         accShapes = {chID: np.shape(MMMArr) for chID, MMMArr in (XformS5E25D40MMMs if xform else S5E25D40MMMs).items()}
         assert accShapes == S5E25D40Shapes
 
     @pytest.mark.parametrize('channel, xform',
-                             [(8, False), (80, False),
-                              (84, False), (20, False), (59, False), (76, False),
+                             [(8, False), (80, False),  # 8, 80 - CDBs are minMeanMax encoded by MCU
+                              (84, False), (20, False), (59, False), (76, False),  # 84, 20, 59, 76 - Not MMM encoded
```

Comment:
Spoke to Pete about the labeling of "MMM-encoded" and "non-encoded"; sticking with documenting it as comments just for future reference